x-pack/plugins/siem/public/store/timeline/model.ts

```diff
@@ -93,7 +93,7 @@ export const timelineDefaults: Readonly<
   isFavorite: false,
   isLive: false,
   itemsPerPage: 25,
-  itemsPerPageOptions: [10, 25, 50],
+  itemsPerPageOptions: [10, 25, 50, 100],
```

Comment:
I assume this is intentful and we want it to include 100. Pointing it out in case you didn't want 100 here for the options.

Reply:
yes, we want to do that